ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
+If you are submitting a bug issue:
+- [ ] Please include your operating system type and version number, as well as your Python, scikit-learn, numpy, and scipy versions. This information can be found by runnning [this code snippet](https://gist.github.com/nelson-liu/108ad4a0571e43ac3d80).
```

Comment:
What about going further and adding a ``sklearn.print_version_info()`` function that will print-out all the info we'd like? We could even make it print things like system info, 32/64-bit, compiler used, whether the installation was done via wheel/conda, etc. Maybe overkill, but could be very useful in the long run.